apps/microtvm/arduino/template_project/microtvm_api_server.py

```diff
@@ -287,11 +288,11 @@ def _find_modified_include_path(self, project_dir, file_path, include_path):
         return include_path
 
     def _get_platform_version(self, arduino_cli_path: str) -> float:
+        # sample output of this command:
+        # 'arduino-cli alpha Version: 0.18.3 Commit: d710b642 Date: 2021-05-14T12:36:58Z\n'
         version_output = subprocess.check_output([arduino_cli_path, "version"], encoding="utf-8")
-        version_output = (
-            version_output.replace("\n", "").replace("\r", "").replace(":", "").lower().split(" ")
-        )
-        full_version = version_output[version_output.index("version") + 1].split(".")
+        full_version = re.findall("version: ([\.0-9]*)", version_output.lower())
```

Comment:
nit: should add `r" string prefix` (see https://docs.python.org/3/reference/lexical_analysis.html#string-and-bytes-literals regarding backslashes)